<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairTemplateDialog</class>
 <widget class="QDialog" name="RepairTemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepairTemplateDialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>模板名</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="templteName">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>6</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>9</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TagName</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tagName">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>5</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="IdName">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>34</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>64</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>64</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
